auth/shared-signal-receiver/handlers/request-handler.ts

```diff
@@ -40,7 +41,10 @@ export const requestHandler = async (
     }
 
     return generateResponse(StatusCodes.BAD_REQUEST, ReasonPhrases.BAD_REQUEST);
-  } catch {
+  } catch (error) {
+    if (error instanceof CognitoError) {
+      return generateResponse(StatusCodes.INTERNAL_SERVER_ERROR, error.message);
+    }
```

Comment:
I think this should be handled at the [root](https://github.com/alphagov/govuk-mobile-backend/blob/main/auth/shared-signal-receiver/app.ts#L42-L47) - having multiple layers of error handling for the same condition might be misleading.